source/documentBase.py

```diff
@@ -4,7 +4,6 @@
 #See the file COPYING for more details.
 
 from baseObject import AutoPropertyObject, ScriptableObject
-from scriptHandler import isScriptWaiting
 import config
 import textInfos
 import speech

```

Comment:
I think you can do the same with `speech`

Reply:
I also moved `ui`